src/client/containers/batch-detail-page/batch-detail-page.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SidebarMenu from '../../components/side-navigation/sidebar.component';
+import AddBatch from '../../components/batch/add-batch.component';
```

Comment:
link should be '../../components/add-batch/add-batch.component'
```suggestion
import AddBatch from '../../components/add-batch/add-batch.component';
```

Reply:
I thought of grouping batch components as a suggestion in the beginning, since we have both `add` and `edit` for `batch` component. but actually we just have all components out on its own, so that's ok I will just change it now.